### PR TITLE
Add a simple download spec and mention save_path in the readme

### PR DESCRIPTION
**Pull request by nickhammond**
For #169 and to add `save_path` to the readme.

### CODE REVIEW

**Comment by route** [comment]
Thank you!

File: spec/download_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+          base_url: Ferrum::Server.server.base_url,
+          save_path: "/tmp/ferrum"
+        )
+      end
+
+      it "saves an attachment" do
+        browser.go_to("/attachment.pdf")
+
+        expect(File.exist?("/tmp/ferrum/attachment.pdf")).to be true
+      end
```
Comment (Mifrill) [inline spec/download_spec.rb:17]: :thinking: weired point is this test failing for me locally:
```
Failures:

  1) Ferrum::Browser download support saves an attachment
     Failure/Error: expect(File.exist?("/tmp/ferrum/attachment.pdf")).to be true
     
       expected true
            got false
     # ./spec/download_spec.rb:16:in `block (3 levels) in <module:Ferrum>'

Finished in 0.49189 seconds (files took 0.57409 seconds to load)
1 example, 1 failure
```
For both ruby versions: 2.7.4 and for 3.0.2.

Comment (route) [reply spec/download_spec.rb:17]: Yea I'll fix

Comment (nickhammond) [reply spec/download_spec.rb:17]: After I prepared this I noticed that Ferrum was really consistent when not using the `url` option but once I used the `url` option with a docker container(browserless/chrome) then it was really inconsistent with downloads and would often hang. 

@Mifrill Any indicators as to why it's failing?

Comment (route) [reply spec/download_spec.rb:17]: @nickhammond  try master branch, is it any better now?

Comment (nickhammond) [reply spec/download_spec.rb:17]: @route I can't seem to get the same behavior from yesterday but it's consistently working when downloading something so should be good. I have a failing spec locally but it's related to window position "allows the window to be positioned" browser_spec:122 so not the same as on the above but no worries.

**Comment by route** [comment]
Yea me too, looks like something broke on Chrome's side... needs investigation

File: spec/download_spec.rb (hunk 1 above)
Comment (route) [reply spec/download_spec.rb:17]: Yea me too, looks like something broke on Chrome's side... needs investigation

**Pull request merged**